app/src/androidTest/java/com/sdpteam/connectout/profileList/ProfileFilterFragmentTest.java

```diff
@@ -65,7 +65,7 @@ public void filteringByNameIsOrdered() {
             }
         });
         if (!profiles.isEmpty()) {
-            List<String> givenList = profiles.stream().map(Profile::getName).collect(Collectors.toList());
+            List<String> givenList = profiles.stream().map(Profile::getNameLowercase).collect(Collectors.toList());
```

Comment:
Great idea to change this! Indeed, it made no sense to consider uppercase and lowercase letters separately